htdocs/js/jquery.imageshrink.js

```diff
@@ -5,9 +5,16 @@
 // - Hiding zoom cursors for images that won't zoom
 // - Marking tall images so they get modified squish behavior
 jQuery(function($) {
+    // Entry-like things that might have images in them we want to possibly shrink. Should be in CSS-selector format
+    const imgHolders = ['.entry-content', '.comment-content', '.InboxItem_Content .Body'];
+    // containers to observe for changes. Should be a plain id name.
+    const observedContainers = ['entries', 'comments', 'inbox_message_list'];
+
+    // This just turns imgHolders into a single CSS query string.
+    const imgQuery = imgHolders.join(" img, ") + " img";
```

Comment:
This is purely 💅 , but could we use the more standard term "selector" in these names instead of "query"?

Reply:
Unrelatedly but much more urgently: Is the JS minimizer used by build-static able to cope with `const` and `let` yet, or does it choke on them and demand `var`? I think it's not used in dev, so you'd have to specifically check somehow.